.cirrus.yml

```diff
@@ -173,6 +178,7 @@ linux_task:
                 cc: gcc-4.9
                 cxx: g++-4.9
                 rust_version: 1.42.0
+                DONT_RUN_CLIPPY: 1
```

Comment:
this should be in `env` block on the same level as `container`

Reply:
Oh >_< Silly me! Thanks, and sorry for wasting your time with a user error.